server/restivus-swagger.js

```diff
@@ -66,12 +67,18 @@ Restivus.prototype.addSwagger = function(path) {
 
         // Add paths to Swagger doc
         _.extend(doc, {"paths": paths});
+
+        // Add definitions
+        if(swagger.definitions !== undefined) {
+          _.extend(doc, {"definitions": swagger.definitions});
+        }
+
         // Return swagger.json
         return doc;
 
       } else {
         // Error handling
-        return {"error": "Swagger metadata not given in Restivus config."};
+        return {"error": "Swagger configuration not given for Restivus."};
```

Comment:
Perhaps move this to the front of the function, so that it is closer to its conditional. E.g.

```js
if ( swagger === undefined || 
    swagger.meta === undefined ) {
        return {"error": "Swagger configuration not given for Restivus."};
};
```